stock_move_line_change_lot/models/stock_move_line.py

```diff
@@ -0,0 +1,166 @@
+# Copyright 2020 Camptocamp SA (http://www.camptocamp.com)
+# Copyright 2024 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, models
+from odoo.exceptions import UserError
+from odoo.tools.float_utils import float_compare
+
+
+def is_lesser(value, other, rounding):
```

Comment:
These functions are great and  more meaningful than float_compare.

They deserve a dedicated module